test/CurveTests.ts

```diff
@@ -0,0 +1,88 @@
+import { ethers } from "hardhat";
+import { expect } from "chai";
+import { helperResetNetwork, helperSwapETHWith3CRV } from "./MainnetHelper";
+import { createMetapool, getMetapool, fundMetapool, setupMetapool, exchangeLvUSDfor3CRV, exchange3CRVfor3LvUSD } from "./CurveHelper";
+import { buildContractTestContext, ContractTestContext } from "./ContractTestContext";
+
+describe("CurveHelper Test Suite", function () {
+    let owner;
+    let user1;
+    let user2;
+    let r: ContractTestContext;
+    let lvUSD;
+    let token3CRV;
+
+    beforeEach(async function () {
+        // Reset network before integration tests
+        helperResetNetwork(14533286);
+        // Setup & deploy contracts
+        r = await buildContractTestContext();
+        lvUSD = r.lvUSD;
+        token3CRV = r.external3CRV;
+        [owner, user1, user2] = await ethers.getSigners();
```

Comment:
You can get those values from r: ContractTestContext, see https://github.com/thisisarchimedes/Archimedes_Finance/blob/29d415134fe09137b73a7cd3a4dd940c76b2d148/test/ContractTestContext.ts#L29